cypress/integration/account_settings/theme_color/custom_colors_sidebarStyles_spec.js

```diff
@@ -0,0 +1,441 @@
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Unread Text icon color change
+        cy.get('.color-icon').eq(4).should('have.css', 'background-color', 'rgb(129, 65, 65)');
+
+        // 5. Save Sidebar Unread Text color change
+        cy.get('#saveSetting').click();
+
+        // 6. Exit user settings
+        cy.get('#accountSettingsHeader > .close').click();
+
+        // * Check Sidebar Unread Text
+        cy.get('.sidebar-item.unread-title').should('have.css', 'color', 'rgb(129, 65, 65)');
+
+        // 7. Open sidebar dropdown
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // 8. Select Account Settings
+        cy.get('#accountSettings').click();
+
+        // 9. Click the Display tab
+        cy.get('#displayButton').click();
+    });
+
+    it('should change Sidebar Text Hover BG color and verify color change', () => {
+        // 1. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 2. Click Sidebar Text Hover BG
+        cy.get('.input-group-addon').eq(5).click();
+
+        // 3. Click on color bar to change color
+        cy.get('.hue-horizontal').click();
+
+        // 4. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Text Hover BG color icon change
+        cy.get('.color-icon').eq(5).should('have.css', 'background-color', 'rgb(69, 191, 191)');
+
+        // 5. Save Sidebar Text Hover BG color change
+        cy.get('#saveSetting').click();
+
+        // * CSS hover is not currently implemented for cypress, unable to find workaround. mouseenter/mouseover does not work with css check
+
+        //  8. Exit user settings
+        // cy.get('#accountSettingsHeader > .close').click();
+
+        // * Check Sidebar Text Hover BG color change
+        // cy.get('#sidebarItem_autem-2').trigger('mouseover, { force: true }');
+        // cy.get('.sidebar-item').should('have.css', 'background-color', 'rgb(69, 191, 191)');
+    });
+
+    it('should change Sidebar Text Active Border color and verify color change', () => {
+        // 1. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 2. Click Sidebar Text Active Border
+        cy.get('.input-group-addon').eq(6).click();
+
+        // 3. Click on color window to change color
+        cy.get('.saturation-black').click();
+
+        // 4. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Text Active Border icon color change
+        cy.get('.color-icon').eq(6).should('have.css', 'background-color', 'rgb(65, 92, 129)');
+
+        // 5. Save Sidebar Text Active Border color change
+        cy.get('#saveSetting').click();
+
+        // unsure how to identify Sidebar Text Active Border color. Commenting out second half of test.
+
```

Comment:
I verified this as well and you're right about the inability to verify this change.  I'd suggest to do the same as I suggested above.

Reply:
removed comments and added `#pasteBox` check